### PR TITLE
Fixes #36866 - Remove table hooks from katello and use foreman

### PR DESCRIPTION
**Pull request by parthaa**
#### What are the changes introduced in this pull request?

Now that TableHooks.js code has been ported to Foreman, we need to have Katello consume it from there so code isn't duplicated. This PR does the clean up / walden job 

#### What are the testing steps for this pull request?

* Check out this PR
* Try some pages that use Table hooks. 
* Make sure they look ok

### CODE REVIEW

**Comment by jeremylenz** [comment]
@parthaa Do we know if Foreman TableHooks has diverged at all from Katello?

**Comment by chris1984** [comment]
@jeremylenz do you mind reviewing the code, while I test the pages to make sure there are not any breakages?

File: webpack/components/extensions/HostDetails/Tabs/ErrataTab/ErrataTab.js [hunk 1]
```diff
@@ -228,7 +229,7 @@ export const ErrataTab = () => {
   });
 
   const installErrataAction = () => installErrata({
-    hostname, search: fetchBulkParams(),
+    hostname, search: fetchBulkParams({}),
```
Comment (jeremylenz) [inline webpack/components/extensions/HostDetails/Tabs/ErrataTab/ErrataTab.js:232]: The need to pass `{}` here is eliminated in https://github.com/theforeman/foreman/pull/9878, but I suppose it's harmless either way. If that gets merged first, you could remove all of these.

**Review by jeremylenz — COMMENTED** [review]
Code LGTM :+1:

**Comment by chris1984** [comment]
@sjha4 said he would test the CV pages, working on testing the other stuff that was changed

**Review by chris1984 — APPROVED** [review]
ACK works well, all the tables/pages/modals render that I tested

File: webpack/components/extensions/HostDetails/Tabs/ErrataTab/ErrataTab.js (hunk 1 above)
Comment (parthaa) [reply webpack/components/extensions/HostDetails/Tabs/ErrataTab/ErrataTab.js:232]: Agreed!. May be I should wait for your PR before merging

**Pull request merged**